deploy/crds/avo.openshift.io_vpcendpoints.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.16.4
```

Comment:
Let's not include the regenereted `controller-gen` pieces without also bumping Boilerplate to the latest tag. For now just fix the other bug 👍